olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -553,4 +553,5 @@ spec:
       image: registry.svc.ci.openshift.org/openshift/knative-v0.14.2:knative-eventing-channel-dispatcher
     - name: IMAGE_KN_CLI_ARTIFACTS
       image: registry.svc.ci.openshift.org/openshift/knative-v0.14.0:kn-cli-artifacts
+  replaces: serverless-operator.v1.7.2
```

Comment:
Why this? I thought we were using the skipRange instead of this.

Reply:
Eh, precautionary. I can try if it works without it. BRB